classes/class-wc-rest-connect-stripe-account-controller.php

```diff
@@ -1,60 +0,0 @@
-<?php
-
-if ( ! defined( 'ABSPATH' ) ) {
-	exit;
-}
-
-if ( class_exists( 'WC_REST_Connect_Stripe_Account_Controller' ) ) {
-	return;
-}
-
-class WC_REST_Connect_Stripe_Account_Controller extends WC_REST_Connect_Base_Controller {
-	protected $rest_base = 'connect/stripe/account';
```

Comment:
In our `client/extensions/woocommerce/state/sites/settings/stripe-connect-account/actions.js`, we makes call to the stripe end point https://github.com/Automattic/woocommerce-services/blob/develop/client/extensions/woocommerce/state/sites/settings/stripe-connect-account/actions.js#L99

Do we need to remove `createAccount` https://github.com/Automattic/woocommerce-services/blob/develop/client/extensions/woocommerce/state/sites/settings/stripe-connect-account/actions.js#L77 as well?

Reply:
Thanks for digging into this @harriswong!

I removed all action, reducer, and selector code related to Stripe account creation:
https://github.com/Automattic/woocommerce-services/pull/2008/commits/66472de0915afc978734cb262c8ef2c3b77a7934

I also made sure to clean up the tests for these as well (It looks like there is an unrelated failing test for Shipping Rate).

Truthfully, I'm not certain where to look for these changes to manually confirm them on my test site. But I did make sure to run through setting up Stripe via onboarding and verifying settings a few times.

It'd be great if you could let me know where to look for future reference.

Thank you!

Reply:
No problem! Thanks for looking into this! I am not too familiar with the Stripe codes inside WCS as well so I did a little digging. I ran a search on `create_account`. That's where I saw the use of `init_core_wizard_payments_config`. 

Then I saw that we are removing the controller `class-wc-rest-connect-stripe-account-controller.php` so the end point caught my eyes.

Reply:
Gotcha. Thanks for sharing!